packages/interface-adapter/lib/web3-shim.ts

```diff
@@ -74,8 +74,12 @@ export class Web3Shim extends Web3 {
     initInterface(this);
   }
 
-  setNetworkType(networkType: NetworkType) {
+  public setNetworkType(networkType: NetworkType) {
     this.networkType = networkType;
     initInterface(this);
   }
+
+  public getNetworkId() {
+    return this.eth.net.getId();
+  }
```

Comment:
I think it inadvisable to paste this into the shim itself... I would imagine there'd be a Web3InterfaceAdapter that implements `getNetworkId()` and calls out to the shim via Web3.js's interface